src/V3AstNodes.cpp

```diff
@@ -18,18 +18,53 @@
 
 #include "V3EmitCBase.h"
 #include "V3File.h"
+#include "V3Global.h"
 #include "V3Graph.h"
 #include "V3Hasher.h"
 #include "V3PartitionGraph.h"  // Just for mtask dumping
 #include "V3String.h"
 
+static void dumpJsonPtr(std::ostream& os, const std::string& name, const AstNode* const valp,
+                        const std::string& unlink_msg = "UNLINKED");
 #include "V3Ast__gen_impl.h"  // Generated by 'astgen'
 #include "V3Ast__gen_macros.h"  // Generated by 'astgen'
 
 #include <iomanip>
 #include <iterator>
+#include <sstream>
+#include <string>
 #include <vector>
 
+// Shorthands for dumping JSON.
+
+// SQOUT - stream quote - print x surrounded with quotes. Example: cout << SQUOT("something")
+#define SQUOT(x) '"' << (x) << '"'
```

Comment:
Suggest cleaner to read the code if just avoid this define and "inline" it to put quotes where needed.